kickstart/signup-teams.yaml

```
@@ -14,28 +14,34 @@ description: >-
     for the last few years we've only advertised the competition to teams who
     are on our mailing list.
 
+
     To accomplish this, we need to create a signup form to collect the institution
     and team supervisor details. Once we have the details, we need to select the
     teams who will be taking part in the competition.
 
+
     Any teams who are taking part for the first time will need to be allocated
     a TLA, ideally one that doesn't clash with any existing teams.
 
+
     Additionally, as the kits contain potentially dangerous components, many of
     which are not CE marked, we normally ask the team leaders to sign a form
     accepting responsibility for it.
 
+
     We should send (digital) copies of the form to team leaders so that they can
     sign & return the forms ahead of time.
 
+
     Once we have the final plans for Kickstart, we need to send an email to the
     team supervisors so that they can prepare for the event.
 
+
     We should aim to tell them:
-     * ~~What venue to come to~~
-     * What time to arrive/when the livestream starts
-     * That their team members should join the Discord server
-     * To send their signed disclaimer forms so that they can get their kits
+     - ~~What venue to come to~~
```

This reads weird. "The venue" still applies for virtual events
```suggestion
     - The venue
```